internal/exeutils/exeutils_windows.go

```diff
@@ -11,20 +11,29 @@ import (
 
 const Extension = ".exe"
 
-// PathForExecutable returns the first path from the PATH env var for which the executable exists
-func PathForExecutable(executable string) string {
-	candidates := strings.Split(os.Getenv("PATH"), string(os.PathListSeparator))
+// FindExecutableOnOSPath returns the first path from the PATH env var for which the executable exists
+func FindExecutableOnOSPath(executable string) string {
+	return FindExecutableOnPath(executable, os.Getenv("PATH"))
+}
+
+func FindExecutableOnPath(executable, PATH string) string {
+	return findExecutable(executable, PATH, os.Getenv("PATHEXT"))
+}
+
+func findExecutable(executable, PATH, PATHEXT string) string {
+	candidates := strings.Split(PATH, string(os.PathListSeparator))
 	var exts []string // list of extensions to look for
-	pathexts := strings.Split(os.Getenv("PATHEXT"), string(os.PathListSeparator))
+	pathexts := funk.Map(strings.Split(PATHEXT, string(os.PathListSeparator)), strings.ToLower).([]string)
```

Comment:
Does windows really ignore case here?

Reply:
Windows file systems are case insensitive, yes.  Usually `PATHEXT` has all upper case endings like `.COM;.EXE;.BAT`, but of course, lower case endings are accepeted as well.